R-package/vignettes/basic_walkthrough.Rmd

```diff
@@ -62,10 +62,10 @@ X <- data.matrix(bank[, c("age", "balance")])
 fit <- lightgbm(
   data = X
   , label = y
+  , objective = "binary"
```

Comment:
Please revert this vignette change. I support making is possible to pass `objective` as a keyword argument, but not recommending it as the preferred way to specify that parameter.

Reply:
Reverted.